phantomjs/bridge.js

```diff
@@ -9,7 +9,7 @@
 /*global QUnit:true, alert:true*/
 (function (factory) {
   if (typeof define === 'function' && define.amd) {
-    require(['qunit'], factory);
+    define('phantom-bridge', ['qunit'], factory);
```

Comment:
@Arkni , yes, i'm aware of `inject` option and actually currently using it to inject named module as in this PR. All these links you've provided are absolutely valid and helpful. But again, my point is - we can improve out-of-box experience for all devs trying to start using this grunt task. And there are configuration options for experienced devs that need special behavior as @samkirkpatrick does.
When i was trying to start using `grunt-contrib-qunit` to test js app with requirejs AMDs I had to spend a couple of days to investigate situation just to find out it could be 5 min work if there are  enough error messages and bridge injected in a different way.